helmfile/values/gatekeeper/podsecuritypolicies.gotmpl

```diff
@@ -0,0 +1,76 @@
+constraints:
+  ingress-nginx:
+    nginx:
+      labelsSelector:
+          app.kubernetes.io/component: controller
+      allow:
+        allowedCapabilities:
+          - NET_BIND_SERVICE
+        {{- if .Values.ingressNginx.controller.useHostPort }}
+        hostNetwork: true
+        {{- end }}
+        hostNamespace: true
+        privilegedEscalation: true
```

Comment:
Can we finalise how we want the names on keys? Do we want to drop all `allowed` and `allow` given that we are in that map already? Else we should add it back to `allowPrivilegeEscalation`, and can we drop the `d` in that one so it matches what's set in security contexts?

Reply:
I will go through them and match them to the security contexts. So adding `allow` drop the `d`